### PR TITLE
Move to GitHub actions

### PR DESCRIPTION
**Pull request by benwoo1110**
Just a simple `mvn package` build with gh actions to ensure test pass and build succeeds, since now travis.org service has closed.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Maven CI/CD
+
+on:
+  push:
+    branches: [ master ]
```
Comment (dumptruckman) [inline .github/workflows/build.yml:5]: The main branch is `main`.

Comment (benwoo1110) [reply .github/workflows/build.yml:5]: Oh yea right

**Review by dumptruckman — APPROVED** [review]
Let's try it.

**Comment by benwoo1110** [comment]
Looks like it’s working, this PR builded properly with the GH actions

**Comment by benwoo1110** [comment]
Yep it builds!

**Comment by nicegamer7** [comment]
Good stuff!

**Pull request merged**